docker/BuildJekyll.Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM ruby:3.0.0
```

Comment:
Also, I'd change the name around: it's currently backwards from the conventional `Dockerfile.<specifier>`.